modules/sample-windows-workload-deployment.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * windows_container_workloads/scheduling-windows-workloads.adoc
+
+[id="sample-windows-workload-deployment_{context}"]
+= Sample Windows container workload deployment
+
+You can deploy Windows container workloads to your cluster once you have a Windows compute node available.
+
+[NOTE]
+====
+This sample deployment is provided for reference only.
+====
+
+.Example `Service` object
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: win-webserver
+  labels:
+    app: win-webserver
+spec:
+  ports:
+    # the port that this service should serve on
+  - port: 80
+    targetPort: 80
+  selector:
+    app: win-webserver
+  type: LoadBalancer
+----
+
+.Example `Deployment` object
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: win-webserver
+  name: win-webserver
+spec:
+  selector:
+    matchLabels:
+      app: win-webserver
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: win-webserver
+      name: win-webserver
+    spec:
+      tolerations:
+      - key: "os"
+        value: "Windows"
+        Effect: "NoSchedule"
+      containers:
+      - name: windowswebserver
+        image: mcr.microsoft.com/powershell:lts-nanoserver-1809
+        imagePullPolicy: IfNotPresent
+        command:
+        - pwsh.exe
```

Comment:
I'm not sure why use "- pwsh.exe" instead of "- powershell.exe".
When running this deployment, I got following error:
  Warning  Failed     <invalid> (x2 over <invalid>)  kubelet, ip-10-0-141-186.us-east-2.compute.internal  Error: failed to start container "windowswebserver": Error response from daemon: container windowswebserver encountered an error during hcsshim::System::CreateProcess: failure in a Windows system call: The system cannot find the file specified. (0x2) extra info: {"CommandLine":"pwsh.exe -command \"$listener = New-Object System.Net.HttpListener; $listener.Prefixes.Add('http://*:80/'); $listener.Start();Write-Host('Listening at http://*:80/'); while ($listener.IsListening) { $context = $listener.GetContext(); $response = $context.Response; $content='\u003chtml\u003e\u003cbody\u003e\u003cH1\u003eRed Hat OpenShift + Windows Container Workloads\u003c/H1\u003e\u003c/body\u003e\u003c/html\u003e'; $buffer = [System.Text.Encoding]::UTF8.GetBytes($content); $response.ContentLength64 = $buffer.Length; $response.OutputStream.Write($buffer, 0, $buffer.Length); $response.Close(); };\"","User":"ContainerAdministrator","WorkingDirectory":"C:\\","Environment":{"KUBERNETES_PORT":"tcp://172.30.0.1:443","KUBERNETES_PORT_443_TCP":"tcp://172.30.0.1:443","KUBERNETES_PORT_443_TCP_ADDR":"172.30.0.1","KUBERNETES_PORT_443_TCP_PORT":"443","KUBERNETES_PORT_443_TCP_PROTO":"tcp","KUBERNETES_SERVICE_HOST":"172.30.0.1","KUBERNETES_SERVICE_PORT":"443","KUBERNETES_SERVICE_PORT_HTTPS":"443"},"CreateStdInPipe":true,"CreateStdOutPipe":true,"CreateStdErrPipe":true,"ConsoleSize":[0,0]}

Reply:
@aravindhp @gmarkley-VI  two reported suggestions here for the sample deployment. Can you confirm these changes?
1. Replace `image: mcr.microsoft.com/powershell:lts-nanoserver-1809` &rarr; `mcr.microsoft.com/windows/servercore:ltsc2019`
2. Update `- pwsh.exe` &rarr; `- powershell.exe`

Thanks!

Reply:
>Replace image: mcr.microsoft.com/powershell:lts-nanoserver-1809 → mcr.microsoft.com/windows/servercore:ltsc2019

I don't see a reason to make this change. `mcr.microsoft.com/windows/servercore:ltsc2019` is overkill for the example at `Deployment`.

Reply:
@aravindhp does the image usage in the deployment differ from our Windows Server OS support we documented?
>* Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019
>* Windows Server Semi-Annual Channel (SAC): Windows Server ver 1909

Reply:
Ah I see the issue. `1809`. I think I need new glasses :-) I am fine with `mcr.microsoft.com/windows/servercore:ltsc2019` though ideally we should check to see if `mcr.microsoft.com/powershell:lts-nanoserver-1909` and use that here. @sgaoshang have you tried that?

Reply:
@aravindhp Hi, I tried it in Azure, `mcr.microsoft.com/powershell:lts-nanoserver-1909` works with Windows Server ver 1909.
Here's the image I used in machineset:
          image:
            offer: WindowsServer
            publisher: MicrosoftWindowsServer
            resourceID: ""
            sku: datacenter-core-1909-with-containers-smalldisk
            version: latest
But with Windows Server 2019, `mcr.microsoft.com/powershell:lts-nanoserver-1909` won't work.

Reply:
Shouldn`t `lts-nanoserver-1809` work with Windows Server 2019 , since we are using it in our [Network_tests](https://github.com/openshift/windows-machine-config-operator/blob/master/test/e2e/network_test.go#L346) ?

Also `powershell` image uses  [pwsh.exe](https://github.com/PowerShell/PowerShell-Docker/blob/master/release/lts/nanoserver1809/docker/Dockerfile#L89) executable, so the command change is not required

Reply:
@sgaoshang any updates on this sample deployment? I currently have `mcr.microsoft.com/powershell:lts-nanoserver-1909` in the draft doc, but would like confirmation that this is valid.

Reply:
@codyhoag I'm OK with `mcr.microsoft.com/powershell:lts-nanoserver-1909`, thanks.